### PR TITLE
[Modules] Button for decline validation request and send to user reas…

### PR DESCRIPTION
**Pull request by amatosc**
[Modules] Button for decline validation request and send to user reason.

### CODE REVIEW

File: sinfonier-web/app/controllers/Modules.java [hunk 1]
```diff
@@ -502,6 +502,27 @@ public static void privatize(String id, Integer versionCode) throws SinfonierExc
     }
   }
 
+  public static void decline(String id, Integer versionCode, String message) throws SinfonierException {
```
Comment (fandrod) [inline sinfonier-web/app/controllers/Modules.java:507]: This method needs a @Check("ADMIN"). Only superadmin users can execute this action

File: sinfonier-web/app/controllers/Modules.java [hunk 2]
```diff
@@ -502,6 +502,27 @@ public static void privatize(String id, Integer versionCode) throws SinfonierExc
     }
   }
 
+  public static void decline(String id, Integer versionCode, String message) throws SinfonierException {
+    checkAuthenticity();
+    Module module = Module.findById(id);
+
+    if(module == null) {
+      moduleNotFoundError("Impossible set it to privatize because we can't found the id: ", id);
```
Comment (fandrod) [inline sinfonier-web/app/controllers/Modules.java]: This isn't the correct message for this action, maybe copy & paste?

**Review by fandrod — CHANGES_REQUESTED** [review]
Review previous comments

File: sinfonier-web/app/controllers/Modules.java (hunk 1 above)
Comment (amatosc) [reply sinfonier-web/app/controllers/Modules.java:507]: No, any admin that can validate, can decline as well.

**Review by ivan11paths — APPROVED** [review]

**Review by fandrod — APPROVED** [review]

**Pull request merged**